chapters/ch45.md

```diff
@@ -1,38 +1,35 @@
 
---> _replace THIS LINE by your translation for the above line_
+Xem xét một ví dụ nữa. Giả sử bạn đang xây dựng một hệ thống dịch máy từ tiếng Trung sang tiếng Anh. Hệ thống của bạn hoạt động bằng cách nhập một câu tiếng Trung *C* và tính toán một gía trị Điểm<sub>C</sub>(*E*) cho mỗi bản dịch khả dĩ E. Ví dụ, bạn có thể sử dụng Điểm<sub>C</sub>(*E*) = P(*E*|*C*), xác suất dịch được E cho rằng câu đầu vào là *C*.
 
 > Your algorithm translates sentences by trying to compute:
 
---> _replace THIS LINE by your translation for the above line_
+Thuật toán của bạn dịch các câu bằng cách cố gắng tính toán:
 
 ![img](../imgs/C45_01.png)
 
-> However, the set of all possible English sentences ​*E* i​ s too large, so you rely on a heuristic search algorithm.
+> However, the set of all possible English sentences *E* is too large, so you rely on a heuristic search algorithm.
 
---> _replace THIS LINE by your translation for the above line_
+Tuy nhiên, tập hợp các câu tiếng Anh có thể *E* quá lớn nên bạn dựa vào thuật toán tìm kiếm thực nghiệm.
 
-> Suppose your algorithm outputs an incorrect translation ​*E​*<sub>out</sub>​ rather than some correct translation ​E​*. Then the Optimization Verification test would ask you to compute whether Score​<sub>C</sub>​(*E**) > Score​<sub>C</sub>​(*E*<sub>out</sub>). If this inequality holds, then the Score​<sub>C</sub>​(.) correctly recognized E* as a superior output to *E​*<sub>out</sub>​; thus, you would attribute this error to the approximate search algorithm. Otherwise, you attribute this error to the computation of Score​<sub>C</sub>​(.).
+> Suppose your algorithm outputs an incorrect translation ​*E​*<sub>out</sub>​ rather than some correct translation ​E​*. Then the Optimization Verification test would ask you to compute whether Score​<sub>C</sub>​(*E*\*) > Score​<sub>C</sub>​(*E*<sub>out</sub>). If this inequality holds, then the Score​<sub>C</sub>​(.) correctly recognized E\* as a superior output to *E​*<sub>out</sub>​; thus, you would attribute this error to the approximate search algorithm. Otherwise, you attribute this error to the computation of Score​<sub>C</sub>​(.).
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử thuật toán của bạn dịch ra một bản không chính xác *E*<sub>out</sub> thay vì một bản dịch chính xác E\* nào đó. Sau đó, bài kiểm tra Xác minh tối ưu sẽ yêu cầu bạn tính toán xem liệu Điểm<sub>C</sub>(*E\**) > Điểm<sub>C</sub>(*E*<sub>out</sub>) hay không. Nếu bất đẳng thức này đúng thì cách tính Điểm<sub>C</sub>(.) đã nhận dạng chính xác E\* có điểm cao hơn so với *E*<sub>out</sub>; do đó, bạn sẽ gán lỗi này cho thuật toán tìm kiếm gần đúng. Ngược lại, bạn gán lỗi này cho cách tính Điểm<sub>C</sub>(.)
```

Comment:
- Nên dịch "Then" là "Thì"

Reply:
- xem liệu Điểm<sub>C</sub>(*E\**) > Điểm<sub>C</sub>(*E*<sub>out</sub>) **_có đúng_** hay không

Reply:
- Công thức ở đây nhìn hơi lạ. Mình không biết bản gốc có bị để nhầm không.

Reply:
- Nếu bất đẳng thức này đúng thì cách tính Điểm<sub>C</sub>(.) đã nhận dạng chính xác E\* ~~có điểm cao~~ **_tốt_** hơn so với *E*<sub>out</sub>;
- do đó, bạn sẽ ~~gán~~ **_coi_** lỗi này ~~cho~~ **_là do_** thuật toán tìm kiếm gần đúng. 
- Ngược lại, bạn ~~gán~~ **_coi_** lỗi này ~~cho~~ **_thuộc về_** cách tính Điểm<sub>C</sub>(.)

Reply:
E kiểm tra lại so với bản gốc thì công thức đúng rồi ạ

Reply:
@ngcthuong lạ như thế nào nhỉ?
@tiena2cva em fix/phản hồi mấy comment của @ngcthuong rồi 'mark this as resolve' nhé.